modules/v2/metadata.display.yaml

```diff
@@ -115,6 +115,14 @@ spec:
         vpc_access:
           name: vpc_access
           title: Vpc Access
+          alt_defaults:
+            - type: ALTERNATE_TYPE_DC
+              value:
```

Comment:
We need the value to be yaml equivalent of `{ network_interfaces : { network : "default", subnetwork : "default" } }`.   I think it should be
```
          alt_defaults:
            - type: ALTERNATE_TYPE_DC
              value:
                network_interfaces:
                  network: default
                  subnetwork: default
```

Reply:
sgtm